scene/resources/audio_stream_wav.cpp

```diff
@@ -526,16 +526,15 @@ Vector<uint8_t> AudioStreamWAV::get_data() const {
 
 Error AudioStreamWAV::save_to_wav(const String &p_path) {
 	if (format == AudioStreamWAV::FORMAT_IMA_ADPCM || format == AudioStreamWAV::FORMAT_QOA) {
-		WARN_PRINT("Saving IMA_ADPCM and QOA samples is not supported yet");
-		return ERR_UNAVAILABLE;
+		ERR_FAIL_V_MSG(ERR_UNAVAILABLE, "Saving IMA ADPCM or Quite OK Audio samples is not supported. Quite OK Audio data can still be saved as a .qoa file using FileAccess.");
```

Comment:
Does saving a QOA file using `ResourceSaver.save()` actually write a standard `.qoa` file that audio players will accept?

As I understand it, FileAccess cannot write the QOA file unless you manually reimplement the header writing, then write the packed data after it. It is technically possible to do, but directing users to FileAccess when they basically have to reimplement format saving may be misleading.

Reply:
Unlike IMA ADPCM, the QOA implementation within Godot actually creates full QOA file data including the header, therefore the following should be valid:
```gdscript
var f := FileAccess.open("path/to/file.qoa", FileAccess.WRITE)
    f.store_buffer(my_qoa_audiostream.data)
```